xbmc/pvr/PVRPlaybackState.cpp

```diff
@@ -363,6 +363,13 @@ void CPVRPlaybackState::StartPlayback(
         client->StreamClosed();
 
       client->GetChannelStreamProperties(item->GetPVRChannelInfoTag(), props, fromEpgAsLive);
+
+      if (props.LivePlaybackAsEPG())
+      {
+        std::shared_ptr<CPVREpgInfoTag> epgTag = item->GetPVRChannelInfoTag()->GetEPGNow();
```

Comment:
this could be `const std:.shared_ptr...`

Reply:
Done